inbm/dispatcher-agent/tests/unit/schedule/test_sqlite_manager.py

```diff
@@ -0,0 +1,100 @@
+from unittest import TestCase
+
+from dispatcher.schedule.sqlite_manager import SqliteManager
+from dispatcher.schedule.schedules import SingleSchedule, RepeatedSchedule
+from dispatcher.dispatcher_exception import DispatcherException
+
+class TestSqliteManager(TestCase):
+    
+    def setUp(self) -> None:
+        self.db = SqliteManager(":memory:")
```

Comment:
<h3>$\color{#15F4EE}Miscellaneous$</h3>

Consider using a dedicated test database configuration or mocking database interactions to avoid potential side effects on the actual database.

Reply:
sqlite3 has an in-memory DB that can be used for testing.